src/ng/compile.js

```diff
@@ -3032,6 +3053,29 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
             removeWatchCollection.push(removeWatch);
             break;
 
+          case '<':
+            if (!hasOwnProperty.call(attrs, attrName)) {
+              if (optional) break;
+              attrs[attrName] = void 0;
+            }
+            if (optional && !attrs[attrName]) break;
+
+            parentGet = $parse(attrs[attrName]);
+
+            destination[scopeName] = parentGet(scope);
+
+            if (definition.collection) {
+              removeWatch = scope.$watchCollection(parentGet, function parentCollectionValueWatchAction(newParentValue) {
```

Comment:
I just found a use case for this today while using a 2-way binding (and wishing it was a 1-way binding)... when doing things like `<element binding="array | orderBy:'foo'">` using a `=` binding causes an infinite digest while using `=*` works. But `=*` or `<*` really doesn't seem like a great solution for that, it would be great if there was some other way. Maybe things like `filter` and `orderby` should somehow return the same instance each time if it hasn't changed?